tut/getting-started.md

```diff
@@ -177,16 +179,18 @@ Complete the following steps to create an application from an APK:
 
 1. Create a folder for your application (for example, `my-application`) and place your APK inside this folder.
 
-2. Create a `manifest.yaml` file in that folder. The manifest contains the name and [instance type](https://discourse.ubuntu.com/t/application-manifest/24197#instance-type) for the application. Choose an instance type that is suitable for your application. If your instance is equipped with a GPU and your application requires the use of the GPU for rendering and video encoding, select an instance type with GPU support like `g4.3`. For other instance types, the container will use a GPU if available or software encoding otherwise.
+2. Create a `manifest.yaml` file in that folder. The manifest contains the name and if you want to define a resource preset, the resources for the application. If your instance is equipped with a GPU and your application requires the use of the GPU for rendering and video encoding, make sure to mention the GPU requirement using the `resources` attribute. Otherwise, the container will use a GPU if available or software encoding.
 
    For example, the file could look like this:
 
    ```yaml
    name: my-application
-   instance-type: a4.3
+   resources:
+     cpus: 6
```

Comment:
If people create an AMS application with resources preset, the following attributes must be set
- cpus
- memory
- disk-size

In the above case, the `memory` is missing.

Reply:
Added now